app/code/Magento/Catalog/view/frontend/templates/product/listing.phtml

```diff
@@ -16,7 +16,6 @@
  */
 ?>
 <?php
-$start = microtime(true);
```

Comment:
Please, remove this change from the current commit. I believe you have already introduced the same change in another PR.

Thank you.